### ISSUE TITLE
Requests: New game suggestions

### ISSUE THREAD

**Issue by bscongluanbui** [thebrumby/HotWalletClaimer#34]
Hi admin. A new game of Sei Network release. I think it potential with over 70k  group member a few days.
It Spell bot, collect mana points and upgrade
https://t.me/spell_wallet_bot/wallet?startapp=referral=sei1pzs0eqkmw9jkda3ar79fg0dtky4fukag3nh39p
I hope you considered it in the next game.
Thank for your effort

**Comment by bscongluanbui** [comment]
Hello another game with Over Millions user and Ton watched. PocketFi bot
https://t.me/pocketfi_bot
Hope you consider it.

**Comment by thebrumby** [comment]
> Hello another game with Over Millions user and Ton watched. PocketFi bot https://t.me/pocketfi_bot Hope you consider it.

Hi,

PocketFi has now been added to the repository. Because of its unusual timers which burn your reward if you fail to claim in time, the default is it will make 4-hourly claims. 

`./launch.sh pocketfi`

This will likely to the last game for this week due to time constraints :)

**Comment by bscongluanbui** [comment]
> > Hello another game with Over Millions user and Ton watched. PocketFi bot https://t.me/pocketfi_bot Hope you consider it.
> 
> 
> 
> Hi,
> 
> 
> 
> PocketFi has now been added to the repository. Because of its unusual timers which burn your reward if you fail to claim in time, the default is it will make 4-hourly claims. 
> 
> 
> 
> `./launch.sh pocketfi`
> 
> 
> 
> This will likely to the last game for this week due to time constraints :) 

Thank you

**Comment by Joelpham24** [comment]
Hello sir a new Bot Telegram game from FTN Network , this game have a million users https://t.me/vivaftn_bot/
Hope you consider it.

**Comment by thebrumby** [comment]
> Hello sir a new Bot Telegram game from FTN Network , this game have a million users https://t.me/vivaftn_bot/ Hope you consider it.

Thanks for the suggestion, Joel. We'll take a look at what's required to add that in the coming days. We have a limit of 1 to 2 games a week due to the time needed plus testing, but will see what's involved.

**Comment by bscongluanbui** [comment]
Hi another game with over 3 millions people is CEXIO bot https://t.me/cexio_tap_bot. Claim mode is every 4 hour.
Can you consider it on the next week?
Best Regards

**Comment by sonnguyenHT** [comment]
Hi, @thebrumby, have another game https://t.me/hamster_kombat_bot , the game play is little bit different but I think it is a oke gamefi

**Comment by bscongluanbui** [comment]
Hi https://t.me/TimeFarmCryptoBot. Time Farm is another good project. Pls consider it on the future

**Comment by thebrumby** [comment]
> Hello sir a new Bot Telegram game from FTN Network , this game have a million users https://t.me// Hope you consider it.

Hey @Joelpham24,

We looked at the game today, but unfortunately, the game developers have it tightly locked down so it doesn't work in Telegram Web (it works on mobile phones only). We tried our usual tricks to get around this, but for now, we haven't been able to bypass it.

We'll move on for now but may revisit it at a later date.

Thanks for the suggestion, and sorry we can't implement it yet.

**Comment by thebrumby** [comment]
> Hi another game with over 3 millions people is CEXIO bot https://t.me/cexio_tap_bot. Claim mode is every 4 hour. Can you consider it on the next week? Best Regards

Hi @bscongluanbui,

This also seems to be blocked on Telegram Web?

**Comment by thebrumby** [comment]
> Hi, @thebrumby, have another game https://t.me/hamster_kombat_bot , the game play is little bit different but I think it is a oke gamefi

Hi @sonnguyenHT,

Looks like this game has the same issue as the other 2 games. The developers blocked access via Telegram Web, so we are not able to proceed at this time. Sorry about that.

**Comment by thebrumby** [comment]
> Hi https://t.me/TimeFarmCryptoBot. Time Farm is another good project. Pls consider it on the future

Hi, 

We will take a look shortly regarding this integration. It looks possible :)

**Comment by bscongluanbui** [comment]
> > Hi another game with over 3 millions people is CEXIO bot https://t.me/cexio_tap_bot. Claim mode is every 4 hour. Can you consider it on the next week? Best Regards
> 
> Hi @bscongluanbui,
> 
> This also seems to be blocked on Telegram Web?

Hi have a extension on chrome can by bass that
https://drive.google.com/file/d/1RnTjBQ1IrH2jXiZP9qiiepyXI_hZmDsj/view?usp=drivesdk
can you consider it

**Comment by thebrumby** [comment]
Hi, that download link is not working for me (CEX ID missing). If we can find out what manipulation they are making we will attempt to recreate it. Thanks.

**Comment by bscongluanbui** [comment]
> Hi, that download link is not working for me (CEX ID missing). If we can find out what manipulation they are making we will attempt to recreate it. Thanks.

https://drive.google.com/file/d/1QJbeFpXMIt_WA3PKdwjiFVo_NhB7NsIh/view?usp=sharing
I fix the link. It's working on cent browser or chromium. thank you

**Comment by bulat-gab** [comment]
Hi, this game looks quite simple: https://t.me/fueljetton_bot. Need to click every 12h (similar to Blum) and click to watch ads occasionally. There are no mini games at the moment.

![image](https://github.com/thebrumby/HotWalletClaimer/assets/14952031/c9b243ca-f4ee-475f-8543-704053be1b71)

**Comment by thebrumby** [comment]
> Hi, this game looks quite simple: https://t.me/fueljetton_bot. Need to click every 12h (similar to Blum) and click to watch ads occasionally. There are no mini games at the moment.

Hi, 

Thanks for the recommendation. We have added $Fuel to the repository which can be added to the claim bot using 
```
./launch.sh fuel
```

As always, you need to have progressed to making at least one claim manually, the script should then be able to take over and continue claiming. Once the claim is complete, it will also attempt to watch an advert (if available). Further refinement might be needed in the coming days, but in our overnight tests, it claimed the main function on all accounts.

Kind regards.

**Comment by thebrumby** [comment]
Hi everyone, 

Here comes the last game integration of the week! We added Time Farm, which appears to be claiming successfully on our test rigs.

After reading the initial screens, you can hand over to the script to continue claiming. 

```
./launch.sh time-farm
```

Happy claiming :)

**Comment by bscongluanbui** [comment]
Hi sir, A new game support by BeraChain and very pretty games. 
https://t.me/BerasigWallet_bot
The game doesn't support the web but I got the trick to access it. Open game on web--> inspect --> element tab --> Ctr+shift+C and click the pannel game, in Inframe line, click the link and can access the game.
Its a potential game, Hope you can consider on the Future

**Comment by thebrumby** [comment]
> Hi sir, A new game support by BeraChain and very pretty games. https://t.me/BerasigWallet_bot The game doesn't support the web but I got the trick to access it. Open game on web--> inspect --> element tab --> Ctr+shift+C and click the pannel game, in Inframe line, click the link and can access the game. Its a potential game, Hope you can consider on the Future

Thanks for the information :) The next game addition is unlikely to be before next weekend, and we have some great candidates and look forward to adding the next ones.

**Comment by goncharovnr** [comment]
hi, another game on TON with NFT at the price of 77 ton  https://t.me/LumCity_bot

**Comment by Wellynounet** [comment]
> hi, another game on TON with NFT at the price of 77 ton https://t.me/LumCity_bot

we need code to gain acces  or we need waiting some time 

![image](https://github.com/thebrumby/HotWalletClaimer/assets/88185485/cfe8f8a4-0e03-4251-9ea0-3882ddfc8cc2)

**Comment by thebrumby** [comment]
> hi, another game on TON with NFT at the price of 77 ton https://t.me/LumCity_bot

Hi, 

We just added Lum City! As always, the user must log on and make the initial claim. Once this is done, they can hand over to the script for the hourly claims, offset by your settings for the random timer if active. 

```
./launch.sh lumcity
```

**Comment by thebrumby** [comment]
Additionally, you can now do:

```
./launch.sh lumcity-autoupgrade
```

To automatically upgrade the pickaxe (mining speed), although the cost of this will come from the tokens that you already mined.

**Comment by Wellynounet** [comment]
> > ```
> > ./launch.sh lumcity-autoupgrade
> > ```
> 
> Hi can you consider about auto upgrade level of HOT wallet when have enough point?

Probably not because we are already maxed out :)

**Comment by thebrumby** [comment]
> Hi can you consider about auto upgrade level of HOT wallet when have enough point?

Hi,

We don’t intend to add auto-upgrade to HOT. The auto-upgrade games have two possible upgrade paths, mining speed and filling time. 

Non of the auto-upgrade games have no gas fees so, it makes sense just click upgrade mining speed (if possible). With HOT there are dozens of possible strategies which you will upgrade, in what order, and to what level. So it’s for the individual to make the choice. 

Thanks

**Comment by thebrumby** [comment]
We have another game to add to our list, thanks to a recommendation by our Patreon supporter Michel.

Hold wallet, Diamond Hands is a mining-based game that associates itself with the BNB network. Although it currently operates off-chain with no requirement for gas fees.

Users can launch the game by typing `./launch.sh diamond` after making the usual initial first claim. The system requires a seed phrase to log in, so this will need to be entered during the setup phase to allow the script to take over.

**Comment by mexxuz** [comment]
Hi. I see that you have quite a few HOT wallet apps here and I was really hoping that Pixel Wallet would be among them. Unfortunately it's not here. Quite a serious project that you might also consider adding, because it is identical in mechanics and it mines two tokens at once in one wallet. https://t.me/pixel_wallet_bot

**Comment by thebrumby** [comment]
> Hi. I see that you have quite a few HOT wallet apps here and I was really hoping that Pixel Wallet would be among them. Unfortunately it's not here. Quite a serious project that you might also consider adding, because it is identical in mechanics and it mines two tokens at once in one wallet. https://t.me/pixel_wallet_bot

Hi,

The main issue for us with adding new games is time. This script is maintained as a part-time hobby, and mapping a game and testing can take around 4 - 6 hours, or longer if we include auto-upgrade of mining speed etc. 

We currently have a list of game suggestions here and in Patreon, and this one has already been mentioned, so we will consider it along with the others. 

Thanks.

**Comment by mexxuz** [comment]
> > Hi. I see that you have quite a few HOT wallet apps here and I was really hoping that Pixel Wallet would be among them. Unfortunately it's not here. Quite a serious project that you might also consider adding, because it is identical in mechanics and it mines two tokens at once in one wallet. https://t.me/pixel_wallet_bot
> 
> Hi,
> 
> The main issue for us with adding new games is time. This script is maintained as a part-time hobby, and mapping a game and testing can take around 4 - 6 hours, or longer if we include auto-upgrade of mining speed etc.
> 
> We currently have a list of game suggestions here and in Patreon, and this one has already been mentioned, so we will consider it along with the others.
> 
> Thanks.

Sure, no problem. I realize this is pure enthusiasm. Where can I ask regarding how Time farm works? I've started like other projects, but I'm not getting the clammer to work.

**Comment by goncharovnr** [comment]
Hi, another game https://t.me/MomoAI_bot tokens are immediately withdrawn to the Solana wallet; all you have to do is wait for listing

**Comment by thebrumby** [comment]
> Sure, no problem. I realise this is pure enthusiasm. Where can I ask about how Time Farm and Fuel work? I've started like other projects, but I can't get the clammer to work. Perhaps some special settings are needed?

Hi @mexxuz,

Don't apologize, it's always nice to have enthusiastic people around. There is a list of games in a table on the homepage, and unfortunately Fuel is known not to be working with the system - Hopefully in the coming days we can find a way around it.

Time Farm should be working as expected though, as always, once you have completed the first manual claim, you should be able to pass over to the script. Feel free to share any errors you are seeing. There might be some benefit to deleting your current Time Farm sessions:
```
./launch.sh status
```
```
delete Time
```

**Comment by mexxuz** [comment]
> > Конечно, без проблем. Я понимаю, что это чистый энтузиазм. Где я могу спросить о том, как работают Time Farm и Fuel? Я начинал так же, как и другие проекты, но я не могу заставить clammer работать. Возможно, нужны какие-то особые настройки?
> 
> Здравствуйте,
> 
> Не извиняйтесь, всегда приятно, когда рядом есть увлеченные люди. На главной странице есть список игр в таблице, и, к сожалению, известно, что Fuel не работает с системой - надеюсь, в ближайшие дни мы сможем найти способ обойти это.
> 
> ![образ](https://private-user-images.githubusercontent.com/29182343/344313745-b729104a-c3e5-4127-81d3-f7e5fa26ddaf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2MjEzMzQsIm5iZiI6MTcxOTYyMTAzNCwicGF0aCI6Ii8yOTE4MjM0My8zNDQzMTM3NDUtYjcyOTEwNGEtYzNlNS00MTI3LTgxZDMtZjdlNWZhMjZkZGFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI5VDAwMzAzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMyNDc4MTI1NGE1ZjIwNjUxOTJmYjlhYmYwMDc5ZDY3MmJlNTAzMDEzYzczNDUwNmMyNmE1YmQ5ZDIxZjZiMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sO8WZASPX5sSmmuMLz94OfhhCd-lQ1s9FOOhOi_-GF8)
> 
> Ферма времени должна работать должным образом, хотя, как всегда, после того, как вы выполнили первую ручную заявку, вы сможете перейти к скрипту. Не стесняйтесь делиться любыми ошибками, которые вы видите. Удаление текущих сеансов фермы времени может иметь некоторые преимущества:
> 
> ```
> ./launch.sh status
> ```
> 
> ```
> delete Time
> ```

I tried uninstalling start over. But it doesn't seem to see the claim button on Time farm.
CHROME DRIVER INITIALISED: Try not to exit the script before it detaches.
Step 101 - Attempting to check for the start button (should not be present)...
Step 102 - Attempting to open a link for the app...
Step 103 - Attempting to click the 'Launch' button (may not be present)...
Step 104 - Attempting to switch to the app's iFrame...
Step 106 - Attempting to click the 'Start' button (may not be present)...
Step 108 - Attempting to look for the claim button....
STATUS: The claim button wasn't clickable on this occassion.

Perhaps we should post about this bug in a separate thread?

**Comment by thebrumby** [comment]
Sure, I'll start a new thread.

**Comment by bscongluanbui** [comment]
Hi my friends. A new game Tabizoo bot https://t.me/tabizoobot is a new game. I hope you can add this game on future. Thank you

**Comment by rostamzad1900** [comment]
hi phil
Gamee was updated to another level
check it out.
new mining stage was began.
![Screenshot_20240726-022742](https://github.com/user-attachments/assets/32ccbd89-8a27-4f64-bc38-100d28fd110d)

**Comment by thebrumby** [comment]
> hi phil Gamee was updated to another level check it out. new mining stage was began. !

Hi @rostamzad1900 , 

Unfortunately, the script is not sufficiently advanced to play video-style games, it can't play the game on Blum for the same reason. Ultimately, this script maybe get retired, as they isn't many useful events it can monitor for. 

Thanks

**Comment by phamvanhanh6720** [comment]
Hi @thebrumby , cryptorank.io recently annouanced a tap-2-earn bot https://t.me/cryptorank_app_bot. Initially, it aquire a lot of users and is similar with the Blum bot. I hope you can add auto claming script for this bot

**Comment by thebrumby** [comment]
Thanks for the suggestion. We have two new games already in the pipeline, then we'll take a look afterward.

**Comment by phamvanhanh6720** [comment]
Hi @thebrumby ,
I just found some javascripts code to automatically play the Blum game.
https://github.com/mudachyo/Blum/blob/main/README-EN.md
Can we add this kind of js code with our selenium to play game automatically?

**Comment by thebrumby** [comment]
> Hi @thebrumby , I just found some javascripts code to automatically play the Blum game. https://github.com/mudachyo/Blum/blob/main/README-EN.md Can we add this kind of js code with our selenium to play game automatically?

I took a quick look, but Selenium and Python aren't ideally suited for playing this kind of game. Even if we did get some version working, the response times would be so slow as to make the score very low.

**Comment by thebrumby** [comment]
> Hi @thebrumby , cryptorank.io recently annouanced a tap-2-earn bot https://t.me/cryptorank_app_bot. Initially, it aquire a lot of users and is similar with the Blum bot. I hope you can add auto claming script for this bot

Hi @phamvanhanh6720, 

This game looks OK and should be quite easy to convert to our system, I've made a start now, and will release after tested is complete.
